### DISCUSSION TITLE
⎆ Welcome & Introduction Thread

### DISCUSSION THREAD

**Discussion by DanM3rcurius** [SecretDecks/Documentation#2] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by DanM3rcurius** [comment]
@evolayak @tamagit80 - how do you like the "discussions" feature?

**Comment by evolayak** [comment]
helllloooo, @tamagit80 @bUxEE @stephen-rowan (and @DanM3rcurius ofc) 

I'm Heather- artist, writer and editor.... initially was just interested to join as an artist creating a tarot deck, but I'll be doing some content editing and writing along the way. And anything else that comes up that I can help with! 

Cheers!

**Comment by DanM3rcurius** [comment]
### Introduction 
My name is Dan. I'm eternal light as You are. Being reflected in a somewhat entrepreneurial spirit. Held positions in IT such as Business analyst, official Scrum Master and Crypto Analyst. Acquiring biotickets by Copywriting and killing time with studying and applying permaculture and the hermetic traditions.

Very happy and honoured to be part of this emerging community 🙏

#### Timezone: UTC +2

**Reply by stephen-rowan** [reply]
Hello everyone,

UTC + 1 (BST)

My name is Stephen and my main interest is philosophy. But I can do other stuff like QA, Documentation, Token design and DAO Governance. I am also involved in various Project Catalyst (Cardano) projects. Looking forward to helping out here.

All the Best

**Comment by DanM3rcurius** [comment]
@SecretDecks/founding-core please introduce your self here 🙏

**Comment by wlstylr** [comment]
Hello,
I'm Tyler.   In the past I have produced documentary films, designed apps and websites, founded companies, left companies, worked as a commercial lobster fisherman, and coached snowboarding.  Currently I'm passionate about exploring new ways to quantify and distribute value associated with creative content through blockchain technology. I can help with product management, ux design, content creation, business development, and anything else that I can find a youtube tutorial for.   

Looking forward to helping out on SecretDecks

**Timezone: UTC -7**

**Comment by evolayak** [comment]
I wrote a brief intro before, but I'd like to expand a bit. This is long, but when I speak off the cuff I sometimes forget things. (TL;DR at the very bottom.)

Hi, I'm Heather. Originally from Seattle by way of Washington DC, I've been in Europe for the last 7 years. I come to this project with zero tech knowledge and a very basic understanding of crypto and NFTs... but with a ton of curiosity and enthusiasm; yay! And also during a time when I'm more or less unemployed and wanting to apply what I already know to something totally new that sparks my curiosity.

Initially I got involved in SecretDecks as an analogue, mixed media artist who was stagnating a bit and looking for projects to spark inspiration... incorporating digital tools was already on my mind and reinterpreting the Tarot is just on the periphery of my comfort zone - so hey, why not at least try? I'm now learning more about Tarot, conceptualising a deck, determining what exactly my limitations are and what kind of collaboration I'm seeking.

My more "professional" background is in political sociology (social movements and collective action), media studies and a bit of political economy. I've worked as a research analyst, writer and most recently an editor (from project management to content, line and copyediting and proofreading)... mainly for NGOs and think tanks. As a total layperson in this field, I'm interested in the broader picture and the impact of crypto, blockchain, autonomous organisations and so on, on the future of work, how we relate to contributions to society and compensation, how we make collective decisions and effect change, and also what this means for creative industries, etc. 

I'll be helping with content writing and editing and hopefully more things as time goes on and I expand my knowledge. Really happy to be part of a community comprised of such smart and creative people!

**Comment by MarsKairos** [comment]
Timezone: UTC+2

TL;DR: I am a crazy Fool/Joker on his path to become Magician. Crazy into magick, psychedelia, social engineering, arts, DJing and bringing & keeping souls together with compassion.

Salute brethren and sisters! I am Mars a friend of Dan's since he joined me more than a year ago in a community project aimed to rally individuals with varied interests in mind enhancing topics such as psychedelia, bio - neurohacking, philosophy, spirituality, cutting edge technology and also as of lately crypto and blockchain. The latter led us rapidly into another joint journey that was forged in a form of a Discord community education project aimed at enabling interested parties in general cryptoliteracy (me included). At this point I fully realized his capabilities, vast knowledge and willingness to share his wisdom about anything blockchain related. 

At that time I was introduced to SD project that was just emerging from its embryonic phase. It was double that exciting news as Dan revealed his eagerness to absorb tarot into his life last summer, when we also shared our deep passion for mysticism and magick. I've been living with a single (Crowley's) tarot deck for more than 15 years, which I purchased in an old antiquity store in Glastonbury during my pilgrimage to famous layline node of Albion, the Tor. Since then, I've only used it sporadically, but that is about to change due to recent developments in life that led to a sense of urgency for spiritual path. 

I chose the role of the master of community as I think I've been led actively to this particular role during the past years, which was mostly aimed at building varied communities. Currently, I am also a one of the founding members of Psychedelic Society Berlin, which share many ethical and structural values and challenges of the horizontal decentralized organization. I see a great potential in this parallel emergence as many of the newly formed psychedelic and blockchain communities shared a lot of common ground. 

Thus my service provided to this glorious project should be viewed as the community master nexus. My soul is yearning to get to know y'all and see how we can build and cultivate together this brave new community based on values dear to our hearts. Bless you and your talents, we shall deliver some magic together!

**Reply by DanM3rcurius** [reply]
Thanks for coming onto GitHub and learning the ropes with us 🙏🏿 🧙🏿

**Comment by DanM3rcurius** [comment]
@SecretDecks/founding-core please add your timezone to your introduction 🙏🏿
